node/src/command.rs

```diff
@@ -60,6 +60,8 @@ impl SubstrateCli for Cli {
 			"" | "local" => Box::new(chain_spec::local_testnet_config()?),
 			// on """release""", replace with  included resource
 			#[cfg(not(feature = "brooklyn"))]
+			"sydney-devnet" => Box::new(chain_spec::sydney_devnet_config()?),
```

Comment:
@obsessed-cake What do you mean by "sydney-devnet"? As basically Sydney is always a public testnet. Just one chain.

Reply:
+1,

Reply:
I decided to store the configuration of chain specification in code for Sydney mainnet.
But I didn't know what network to use for it.

Reply:
@obsessed-cake There will be no Sydney-mainnet. Sydney will always remain one net. But at some moment we'll have Mainnet (without any Sydney name)

Reply:
We can just use "Mainnet" name  if you want to address the upcoming mainnet.